tools/shoestring/README.md

```diff
@@ -257,6 +257,13 @@ When `--purge-harvesters` is set, delegates discovered using old keys will be di
 
 ## Files
 
+### docker-compose-recovery.yaml
+
+The docker-compose-recovery.yaml file is used to recovery your node when there is a `server.lock` or `broker.lock` 
```

Comment:
used to recover~~y~~

but this should be in its own Recovery/Troubleshooting section 

Files section is discussing input files